app/grandchallenge/core/widgets.py

```diff
@@ -92,3 +94,17 @@ def media(self):
                 "vendored/bootstrap/js/bootstrap.bundle.min.js",
             ],
         )
+
+
+class MarkdownEditorFullPageWidget(MarkdownEditorInlineWidget):
+    """Customized MarkdownX widget with side-by-side panes."""
+
+    template_name = "markdownx/full_page_widget.html"
+
+    class Media:
```

Comment:
This way we can inherit the assets from the parent. It's best explained [here in the docs](https://docs.djangoproject.com/en/5.1/topics/forms/media/#extend). The use of the dynamic property in `MarkdownInlineEditor` is not strictly necessary either, I think. It could have been a static definition with `extend = False`. The documentation mentions that "Dynamic properties give you complete control over which files are inherited, and which are not." (However, there is no example given with selective inheritance.)